packages/flutter/lib/src/services/text_input.dart

```diff
@@ -483,6 +486,11 @@ class TextInputConfiguration {
   }) : smartDashesType = smartDashesType ?? (obscureText ? SmartDashesType.disabled : SmartDashesType.enabled),
        smartQuotesType = smartQuotesType ?? (obscureText ? SmartQuotesType.disabled : SmartQuotesType.enabled);
 
+  /// The ID of the view that the text input belongs to.
+  ///
+  /// See: [View] and [FlutterView].
```

Comment:
nit: can you please use the "see also:" style from our style guide for this: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#leave-breadcrumbs-in-the-comments

Reply:
Done